src/main/java/io/github/fablabsmc/fablabs/api/gamerule/v1/FabricGameRuleCategory.java

```diff
@@ -0,0 +1,32 @@
+package io.github.fablabsmc.fablabs.api.gamerule.v1;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.minecraft.text.LiteralText;
+import net.minecraft.text.Text;
+import net.minecraft.util.Formatting;
+
+/**
+ * Utility class for registering GameRule objects with custom categories outside of the seven categories MC provides
+ */
+public class FabricGameRuleCategory implements Comparable<FabricGameRuleCategory>{
```

Comment:
Hmm, is implementing Comparable really necessary? I do know you use a TreeMap in another class. Equality checks should suffice in this case (i.e. implement `toString`).

Reply:
Unfortunately `TreeMap` does require the keys to implement `Comparable`, and using `TreeMap` makes sorting easy enough to warrant doing so.